pkgs/by-name/xp/xpipe/package.nix

```diff
@@ -21,6 +21,9 @@
 , libXrender
 , libXtst
 , libXxf86vm
+, util-linux
+, socat
+, hicolor-icon-theme
```

Comment:
please remove

Reply:
I added these dependencies because the newer version requires them. I thought that was acceptable because these are pretty standard dependencies

Reply:
You import the packages to this expression, but don't make use of it in the derivation. If you want to add these to the package then please also add the to build inputs, or wrap the necessary files